### DISCUSSION TITLE
5.1.2 RC

### DISCUSSION THREAD

**Discussion by diegog-sf** [zino-app/graphql-flutter#1273] [Q&A] [answered]
Hello,
We are eager to try out 5.1.2 but we are still hesitant because of the `-beta` suffix, is there an estimated timeline for a version without the beta suffix? or are the beta releases safe for production?

**Comment by vincenzopalazzo** [comment, marked as answer]
>beta releases safe for production?

Of curse, the beta bugs are fixed as soon as possible, and the bugs found in the official version will go through a new beta life cycle that takes longer. So the beta is just a version with super power, and If I was in you I was running always the beta, because only running the beta can give us feedback on what is broken

>there an estimated timeline 

Before Christmas, I'm finding my Santa Claus Clothes

**Reply by diegog-sf** [reply]
Thank you for your answer @vincenzopalazzo! Looking forward to that Christmass gift!

**Reply by vincenzopalazzo** [reply]
Merry Christmas https://github.com/zino-hofmann/graphql-flutter/releases

**Reply by diegog-sf** [reply]
Sweet Christmas indeed!
